### DISCUSSION TITLE
`subtle` and `default` Badge variants

### DISCUSSION THREAD

**Discussion by BenJeau** [mantinedev/mantine#3421] [Q&A] [answered]
It seems that `subtle` and `default` are valid variants in the sense that they make a difference in the Badge's appearance, but they are excluded from the type. Is there a reason for this? Or those variants really shouldn't be used.

Just asking since `subtle` is useful for me.

Here's a picture of some badges - `filled`, `outline`, `default`, and `subtle`:
<img width="305" alt="Screenshot 2023-01-25 at 9 06 47 AM" src="https://user-images.githubusercontent.com/22248828/214584669-2e55b8b8-3347-4ca7-939c-6c1b2d99b1c3.png">

**Comment by rtivital** [comment, marked as answer]
Subtle variant is designed for interactive elements, for example, buttons. It does not make sense for badge to have this variant, as it does not have hover effects. You can still use it if it works for you.

**Reply by BenJeau** [reply]
On that point, should it be added to the `BadgeVariant` type since it is technically a valid value?

**Reply by rtivital** [reply]
Variants handling will be changed in 6.0, you will be able to define any extra types on MantineProvider, type is not planned to be changed though – you will be able to use predefined variants or any string.

https://v6.mantine.dev/changelog/6-0-0/#variants-and-sizes-on-mantineprovider
